lib/KResponsiveWindowMixin.js

```diff
@@ -65,6 +65,7 @@ const windowListeners = [];
 /* methods */
 
 function windowMetrics() {
+  console.warn('Please Use useKResponsiveWindow instead of KResponsiveWindow Mixin');
```

Comment:
`windowMetrics` is called on each window resize so this warning might get printed repeatedly which would be unnecessary. I'd suggest placing it to `mounted` lifecycle hook instead. Then, it will get printed only when a component using this mixin is created in DOM for the first time.